pipeline/transform/user_interests.py

```diff
@@ -3,16 +3,37 @@
 import prefect
 from prefect import task
 
-from pipeline.types import RawUserInterests, User
+from pipeline.types import Interest, RawUserInterests, User
 
 
 @task
 def augment_users_with_interests(
     users: List[User], raw_interests: RawUserInterests
 ) -> List[User]:
-    # TODO: Implement your transform task
 
     logger = prefect.context.get("logger")
-    logger.info("Chris")
+    logger.info(
+        "Finished issues #103 & #194 (Chris); wrote additional test cases as well"
+    )
+
+    # TODO: Implement your transform task
+
+    for user in users:
+
```

Comment:
Coding style point: we don't need all these extra blank lines. I use blank lines to break up blocks of code to be more readable, but a blank line after every line is usually too much.